<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flvParseClass</class>
 <widget class="QMainWindow" name="flvParseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>flvParse</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QTextEdit" name="hexView">
         <property name="font">
          <font>
           <family>FangSong</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="flvStructTree">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionAll_Tags"/>
    <addaction name="actionVideo_Only"/>
    <addaction name="actionAudio_Only"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDisplay"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionVideo_Only">
   <property name="text">
    <string>Video Only</string>
   </property>
  </action>
  <action name="actionAudio_Only">
   <property name="text">
    <string>Audio Only</string>
   </property>
  </action>
  <action name="actionAll_Tags">
   <property name="text">
    <string>All Tags</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
